README.md

```diff
@@ -42,42 +36,39 @@ The performance of the Criteo DRLM workflow also demonstrates the effectiveness
 
 ### Installation
 
-Prior to installing NVTabular, ensure that you meet the following prerequisites:
+NVTabular requires Python version 3.7+. Additionally, GPU support requires:
 
-* CUDA version 10.1+
-* Python version 3.7+
-* NVIDIA Pascal GPU or later
-
-**NOTE**: NVTabular will only run on Linux. Other operating systems are not currently supported.
+* CUDA version 11.0+
+* NVIDIA Pascal GPU or later (Compute Capability >=6.0)
+* NVIDIA driver 450.80.02+
+* Linux or WSL
 
 #### Installing NVTabular Using Conda
 
 NVTabular can be installed with Anaconda from the ```nvidia``` channel by running the following command:
 
 ```
-conda install -c nvidia -c rapidsai -c numba -c conda-forge nvtabular python=3.7 cudatoolkit=11.0
+conda install -c nvidia -c rapidsai -c numba -c conda-forge nvtabular python=3.7 cudatoolkit=11.2
 ```
 
-If you'd like to create a full conda environment to run the example notebooks, do the following:
+#### Installing NVTabular Using Pip
 
-1. Use the [environment files](https://github.com/NVIDIA/NVTabular/tree/main/conda/environments) that have been provided to install the CUDA Toolkit (11.0 or 11.2).
-2. Clone the NVTabular repo and run the following commands from the root directory:
-   ```
-   conda env create -f=conda/environments/nvtabular_dev_cuda11.2.yml
-   conda activate nvtabular_dev_11.2
-   python -m ipykernel install --user --name=nvt
-   pip install -e .
-   jupyter notebook
-   ```
-   When opening a notebook, be sure to select `nvt` from the `Kernel->Change Kernel` menu.
+NVTabular can be installed with `pip` by running the following command:
+
+```
+pip install nvtabular
+```
```

Comment:
I think we should add that you'll need to install all of your own dependencies if you install this way.